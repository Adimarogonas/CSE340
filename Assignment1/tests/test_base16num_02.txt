1AB1x16 
