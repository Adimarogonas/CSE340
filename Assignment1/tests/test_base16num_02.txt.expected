{1AB1x16 , BASE16NUM , 1}
{ , END_OF_FILE , 3}
